integrations/azure_devops.rst

```diff
@@ -0,0 +1,142 @@
+.. _azure_devops:
+
+
+|azure_logo| Azure DevOps
```

Comment:
Please add "Microsoft", "TFS" and "VSTS" to the title.

Reply:
From [this link](https://docs.microsoft.com/en-us/azure/devops/user-guide/about-azure-devops-services-tfs?view=vsts): _"Azure DevOps Services and Azure DevOps Server were formerly named Visual Studio Team Services (VSTS) and Visual Studio Team Foundation Server (TFS)"_, the title could be (not fair enough):

"Microsoft's Azure DevOps (formerly known as RFS or VSTS)"